### DISCUSSION TITLE
How to train with a dataset similar to Cityscapes

### DISCUSSION THREAD

**Discussion by chandlerbing65nm** [open-mmlab/mmselfsup#671] [General]
Is there a way to train self-supervised models here with a dataset similar to Cityscapes?

Below is an example of Cityscapes dataset. The right image is train image and the left is the annotation.

![image](https://user-images.githubusercontent.com/62779617/212287399-6cc348e5-6658-43bc-ad02-d3dabef8762a.png)

I have RGD remote sensing dataset formatted similar to this where there are training and annotation folder, usually used for semantic segmentation. However, I wanted to train it first in self-supervised method before doing the actual segmentation training in mmsegmentation.

**Comment by fangyixiao18** [comment]
You can try MAE with your datasets, which is a baseline method recently on ImageNet, but not sure about the performance on your own remote sensing dataset.

**Reply by chandlerbing65nm** [reply]
> You can try MAE with your datasets, which is a baseline method recently on ImageNet, but not sure about the performance on your own remote sensing dataset.

You mean this? [Masked Autoencoders Are Scalable Vision Learners](https://github.com/open-mmlab/mmselfsup/tree/master/configs/selfsup/mae)

**Reply by fangyixiao18** [reply]
yes

**Comment by chandlerbing65nm** [comment]
It also says here that I can use `.tif` image format as input:

https://github.com/open-mmlab/mmselfsup/blob/684875d09b68ae654b1d3186190a993274dcc297/mmselfsup/datasets/data_sources/imagenet.py#L71

https://github.com/open-mmlab/mmselfsup/blob/684875d09b68ae654b1d3186190a993274dcc297/mmselfsup/datasets/data_sources/imagenet_21k.py#L36

Does it mean that I can use multi-band images (remote sensing)? For example, image with 4 channels `(R,G,B,NIR)`, saved as `image_0.tif`.
